tests/mkcw.bats

```diff
@@ -42,9 +42,10 @@ function mkcw_check_image() {
   fi
 
   # Clean up.
-  umount "$TEST_SCRATCH_DIR"/mount
-  cryptsetup luksClose "$uuid"
-  buildah umount "$ctrID"
+  run_with_log umount -f "$TEST_SCRATCH_DIR"/mount
+  run_with_log sleep 1
```

Comment:
instead of always adding `sleep 1` which always slows down the test and likely does not guarantee that sleep 1 is enough that should be turned into a loop. Try to luksClose and if it fails with EBUSY sleep 1 and retry a few times in a loop.

Also this really needs a comment why the sleep is needed, yes it is in the commit message but that is an extra step for readers to find, especially if the code in questions moves/is modified again.